### PR TITLE
ign -> gz CLI Migration : gz-msgs

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/739

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -148,8 +148,8 @@ endif()
 set_source_files_properties(${gen_headers} ${gen_sources} ${gen_ruby_scripts}
   PROPERTIES GENERATED TRUE)
 
-message(STATUS "Installing Ruby messages to ${CMAKE_INSTALL_PREFIX}/${IGN_LIB_INSTALL_DIR}/ruby/ignition/${IGN_DESIGNATION}${PROJECT_VERSION_MAJOR}")
-install(FILES ${gen_ruby_scripts} DESTINATION ${CMAKE_INSTALL_PREFIX}/${IGN_LIB_INSTALL_DIR}/ruby/ignition/${IGN_DESIGNATION}${PROJECT_VERSION_MAJOR})
+message(STATUS "Installing Ruby messages to ${CMAKE_INSTALL_PREFIX}/${IGN_LIB_INSTALL_DIR}/ruby/gz/${IGN_DESIGNATION}${PROJECT_VERSION_MAJOR}")
+install(FILES ${gen_ruby_scripts} DESTINATION ${CMAKE_INSTALL_PREFIX}/${IGN_LIB_INSTALL_DIR}/ruby/gz/${IGN_DESIGNATION}${PROJECT_VERSION_MAJOR})
```
Comment (chapulina) [inline src/CMakeLists.txt:152]: These changes aren't related to the CLI. I think it's worth it separating it because there can be unintended consequences

**Review by chapulina — COMMENTED** [review]
We have to remember to rebase-merge this, because there are files being moved.

File: src/CMakeLists.txt (hunk 1 above)
Comment (methylDragon) [reply src/CMakeLists.txt]: :boom: reverted

**Review by chapulina — APPROVED** [review]
`UNIT_gz_TEST` building and passing on Linux and macOS 🚀

**Pull request merged**